### PR TITLE
Add <Provider> to /yourschool

### PR DESCRIPTION
**Pull request by joshlory**
I missed this entry point in PR https://github.com/code-dot-org/code-dot-org/pull/19468.

### CODE REVIEW

**Comment by joshlory** [comment]
+cc @epeach as dev-of-day.

**Comment by joshlory** [comment]
Drew caught this, luckily!

File: apps/src/sites/code.org/pages/public/yourschool.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
 import YourSchool from '@cdo/apps/templates/census2017/YourSchool';
+import {getStore, registerReducers} from '@cdo/apps/redux';
+import isRtl from '@cdo/apps/code-studio/isRtlRedux';
```
Comment (joshlory) [inline apps/src/sites/code.org/pages/public/yourschool.js:7]: @islemaster is it bad practice for me to import this "code-studio" reducer on a code.org page?

Comment (islemaster) [reply apps/src/sites/code.org/pages/public/yourschool.js:7]: It should be fine.  I think we're trying to eliminate that boundary anyway.

**Comment by joshlory** [comment]
CircleCI failure appears to be unrelated.

**Pull request merged**